docs/UsersGuide/dev/xml-specification.md

```diff
@@ -609,7 +612,7 @@ specification.
 | port                       | full            | Specifies the behavior for async ports when the message queue is full. One of *drop*, *block*, or *assert*, where *assert* is the default.                                                                                                                                                                                                                                                                                                                                                                                                                                      |                                                                                                                                                                        |
 | port                       | role            | Specifies the role of the port when the modeler=true.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           |                                                                                                                                                                        |
-| commands                   |                 | Optional. Starts the section where software commands are defined.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               |                                                                                                                                                                        |
```

Comment:
@rohandkn I think this deletion is not intended. Can you revert it?

Reply:
Yes, made the change.